packages/load/tests/loaders/documents/documents-from-glob.spec.ts

```diff
@@ -20,7 +20,7 @@ describe('documentsFromGlob', () => {
         loaders: [new GraphQLFileLoader()],
       });
       expect(result).toHaveLength(1);
-      const expectedFiles = globby.sync(glob);
+      const expectedFiles = globSync(glob.replace(/\\/g, '/'));
```

Comment:
Why do we need to change this? This wasn't an issue before.

Reply:
that's because tinyglobby doesn't support Windows-style paths out of the box yet, but soon.

for context, windows supports both backwards-slash path (legacy) and UNIX ones since a few years

Reply:
Then it is a breaking change, and not sure if we can replace it as this library is used a lot on Windows. :/

Reply:
but it works on windows

windows tests pass

Reply:
I ran the tests locally on Windows and they pass indeed. Under the hood, we're transforming Windows-style paths to Unix-style paths through `unixifyWithDriveDirectory` (that was also done before this PR, but only for one of the loaders, iirc). The change here is just a band-aid for the tests to work properly, as we're calling `globSync` on a glob that didn't go through the "transformation" in the same way that it goes when used in a loader, so we need to "patch it" and change the slashes.

Reply:
Ok ok it is unrelated to that. Now I get it! Sorry, I am confused at first!